### PR TITLE
[BE-3318] Update Iceberg Example to use TPCH SF1 dataset on S3

### PR DESCRIPTION
**Pull request by sahil1105**
(As title)

### CODE REVIEW

File: 06-Bodo-Free-Trial/2-ETL-with-Bodo/1-ETL-iceberg-data lakehouse.ipynb [hunk 1]
```diff
@@ -13,9 +13,9 @@
     "\n",
     "The queries are originally in SQL format and here they are implemented using the pandas API.\n",
     "\n",
-    "By defaults runs use Bodo. Hence, data is distributed in chunks across processes.\n",
+    "The data will be loaded from Iceberg tables that are stored on S3 using a Hadoop Filesystem based Catalog.\n",
     "\n",
-    "**To scale and run your application with multiple nodes you can use [Bodo platform](https://platform.bodo.ai/account/login)**"
+    "By defaults runs use Bodo. Hence, data is distributed in chunks across processes."
```
Comment (ritwika314) [inline 06-Bodo-Free-Trial/2-ETL-with-Bodo/1-ETL-iceberg-data lakehouse.ipynb:18]:
```suggestion
    "By default runs use Bodo. Hence, data is distributed in chunks across processes."

File: 06-Bodo-Free-Trial/2-ETL-with-Bodo/1-ETL-iceberg-data lakehouse.ipynb [hunk 2]
```diff
@@ -162,55 +121,59 @@
     "\n",
     "In this section, we load the data required by the queries in pandas DataFrame.\n",
     "\n",
-    "In read_sql_table  method change the first arguement with the table name. To write the tables to iceberg using EMR , https://github.com/Bodo-inc/examples-dev/blob/EMR_ICEBURG_TEST/emr_iceburg_examples/emr_iceburg_read_write.ipynb . \n"
+    "In read_sql_table  method change the first arguement with the table name. "
```
Comment (ritwika314) [inline 06-Bodo-Free-Trial/2-ETL-with-Bodo/1-ETL-iceberg-data lakehouse.ipynb:124]:
```suggestion
    "In the `read_sql_table`  method, change the first argument with the table name. "

**Review by ritwika314 — CHANGES_REQUESTED** [review]
thank you so much @sahil1105 ! Approved with a couple of very minor comments, would be great if @alifarhid could review.

**Review by ritwika314 — APPROVED** [review]

**Review by alifarhid — APPROVED** [review]
Thank you @sahil1105. Approved, assuming it's been tested e2e on trial size already ;)

**Comment by sahil1105** [comment]
> Thank you @sahil1105. Approved, assuming it's been tested e2e on trial size already ;)

Yes, I tested on dev on 2xc5.2xlarge cluster.

**Pull request merged**